lib/list_versions.go

```diff
@@ -27,16 +27,17 @@ func GetTFList(mirrorURL string, preRelease bool) ([]string, error) {
 	var semver string
 	if preRelease == true {
 		// Getting versions from body; should return match /X.X.X-@/ where X is a number,@ is a word character between a-z or A-Z
-		semver = `\/(\d+\.\d+\.\d+)(-[a-zA-z]+\d*)?\/`
+		semver = `\/(\d+\.\d+\.\d+)(-[a-zA-z]+\d*)?\"`
 	} else if preRelease == false {
 		// Getting versions from body; should return match /X.X.X/ where X is a number
-		semver = `\/(\d+\.\d+\.\d+)\/`
+		// without the ending '"' pre-release folders would be tried and break.
+		semver = `\/(\d+\.\d+\.\d+)\"`
```

Comment:
Ditto, not tested:
```suggestion
		semver = `/(\d+.\d+.\d+)/?"`
```

Reply:
Now works on interactive mode. Thank you.

Reply:
Removed the escape of the double quote and added the optional ending slash. My local tests still work with the additions.